apps/code-examples/src/app/code-examples/indicators/alert/basic/alert-demo.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyAlertHarness } from '@skyux/indicators/testing';
+
+import { AlertDemoComponent } from './alert-demo.component';
+import { AlertDemoModule } from './alert-demo.module';
+
+describe('Basic alert', () => {
+  async function setupTest(days: number) {
+    const fixture = TestBed.createComponent(AlertDemoComponent);
+
+    fixture.componentInstance.days = days;
+    fixture.detectChanges();
```

Comment:
Is this `detectChanges` call required before we create the harness? It's generally good practice to give the change detection responsibility to the consumer of the `setupTest` function (since checking for changes would be considered a side effect of calling the function). This will also let the consumer of the `setupTest` function make any initial changes to the fixture before triggering the first cycle of changes.
```
const { fixture } = await setupTest();
fixture.detectChanges() // <-- consumer makes the first detect changes call.
```

Reply:
Done